js/components/authoring.js

```diff
@@ -54,6 +55,18 @@ export default class Authoring extends PureComponent {
     }
   }
 
+  handleBackButtonClick () {
+    const { step } = this.state
+    const { setOption } = this.props
+    if (step === 2) {
+      this.unloadModel()
+    } else if (step === 3) {
+      setOption('interaction', 'drawContinent')
+      setOption('selectableInteractions', [ 'drawContinent', 'eraseContinent', 'none' ])
+      this.setState({step: 2})
```

Comment:
I guess somewhere else in the code, we call the same set of functions to change step from 1 to 2. We could extract that to something like `setStep2()`, so we don't end up with different UI in the future after going next and back (if you forget to modify one of those sets). I might apply to a few lines in `unloadModel` too.